cves/CVE-2014-9604.yml

```diff
@@ -280,8 +295,8 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: false
+  note: There is no IPC here.
```

Comment:
Why not? Describe where the slices are coming from. Justify why this doesn't involve IPC.